capabilities/connect.capabilities.yml

```diff
@@ -139,6 +139,170 @@ properties:
                 type: string
                 examples:
                 - Ethernet1
+  mclag-capabilities:
+    description: Capabilities of the MC-LAG (Multi-Chassis Link Aggregation) feature in the switch
+    type: object
+    properties:
+      max-mclag-groups:
+        description: Defines the maximum number of MC-LAG groups that can be configured on the switch.
+        type: integer
+      max-ports-per-mclag-group:
+        description: Specifies the maximum number of physical ports that can be part of a single MC-LAG group.
+        type: integer
+      max-vlans-per-mclag-group:
+        description: Indicates the maximum number of VLANs that can be supported within a single MC-LAG group.
+        type: integer
+      peer-link-bandwidth:
```

Comment:
I see nowhere to set this in the schema, is it just for reference? Assume so as you would likely have already physically established the links before we get to this point.